File: Python/Functional-2/no34.py

# 
def no34(strings):
    new_list = []

    for x in strings:
        if len(x) != 3 and len(x) != 4:
            new_list.append(x)

    return new_list

print(no34(["a", "bb", "ccc"]))
print(no34(["a", "bb", "ccc", "dddd"]))
print(no34(["ccc", "dddd", "apple"]))

File: JavaScript/Functional-2/no34.js

// given a list of strings, return a list of the strings, omitting any string length 3 or 4
const no34 = (strings) => {
    const newList = strings.filter(n => n.length < 3 || n.length > 4);

    return newList;
};

console.log(no34(['a', 'bb', 'ccc']));
console.log(no34(['a', 'bb', 'ccc', 'dddd']));
console.log(no34(['ccc', 'dddd', 'apple']));
